packages/ui/src/components/va-data-table/hooks/useSortable.ts

```diff
@@ -107,7 +107,7 @@ export default function useSortable (
   // Sets the clicked heading's column as a one to sort by and toggles the sorting order from "asc" to "desc" to `null`
   // (un-sorted) if the same column is clicked again or sets sorting order to "asc" if some other column is chosen.
   function toggleSorting (column: DataTableColumnInternal) {
-    if (column.key === sortBySync.value) {
+    if (column.name === sortBySync.value) {
```

Comment:
name is marked as optional. What if it's not provided? Why is `key` subtituted with `name` everywhere?

Reply:
Here we have two types: column and internal column. Internal column means that it is a built column. If name is not provided, then key will be used.